packages/calypso-build/webpack.config.js

```diff
@@ -43,7 +43,7 @@ const isDevelopment = process.env.NODE_ENV !== 'production';
  * @return {object}                                webpack config
  */
 function getWebpackConfig(
-	env,
+	env = {}, // eslint-disable-line no-unused-vars
```

Comment:
I thought eslint was smart enough by now to not complain about unused args preceding ones that are being used :roll_eyes: Can't find the reference I was thinking of though right now.